overrides/assets/stylesheets/home.css

```diff
@@ -49,6 +49,10 @@
   z-index: -1;
   padding: 20px;
 }
+.animated-gradient img {
+

```

Comment:
looks like an extra newline crept in here?

Reply:
oh no! that was my fault in the github suggestion - sorry 😬. Still can we drop it, and then this looks good